providers/network/resources/tlsshake/tlsshake.go

```diff
@@ -309,12 +309,24 @@ func (s *Tester) parseAlert(data []byte, conf *ScanConfig) error {
 	return nil
 }
 
+var helloRetryRequestRandom = []byte{
```

Comment:
nit: Perhaps add a comment or rename it.

Only the comment in the PR helped me understand that this is not random data, but the random message in the response.

Reply:
Added a comment